.storybook/preview.tsx

```diff
@@ -78,7 +78,6 @@ function makeViewports() {
 }
 
 const parameters = {
-	actions: { argTypesRegex: '^on[A-Z].*' },
 	controls: {
```

Comment:
I would add this too (but add it alphabetically):

```
options: {
		storySort: {
			method: 'alphabetical',
		},
	},
```

This doesn't maintain the original, but it does make it alphabetical which actually leads to a nice side effect, the DAFF logo is the first story to be shown to users 
<img width="965" alt="image" src="https://github.com/user-attachments/assets/ce419ff5-bd7e-4635-b4bb-4cbdd3801db8">

Reply:
- [ ] Sort stories